configure.ac

```diff
@@ -432,8 +432,30 @@ OMRCFG_DEFINE_FLAG([OMR_ENV_DATA64],[],
 	)]
 )
 
-OMRCFG_DEFINE_FLAG_OFF([OMR_GC_COMPRESSED_POINTERS])
-OMRCFG_DEFINE_FLAG_OFF([OMR_GC_FULL_POINTERS])
+# TODO These options are no longer used, and will be deleted.  User's should switch to use OMR_GC_POINTER_MODE.
+AC_ARG_ENABLE([OMR_GC_COMPRESSED_POINTERS],
+    AS_HELP_STRING([--enable-OMR_GC_COMPRESSED_POINTERS], [Enable OMR_GC_COMPRESSED_POINTERS]))
+AC_ARG_ENABLE([OMR_GC_FULL_POINTERS],
+    AS_HELP_STRING([--enable-OMR_GC_FULL_POINTERS], [Enable OMR_GC_FULL_POINTERS]))
+
```

Comment:
@rwy0717 This part should let the configure script continue to accept these options, while ignoring what they are set to. What do you think of this?

Reply:
Sounds good. The doc string should probably say "DEPRECATED" or something.

Reply:
You're using space to indent these options, should be tabs.